src/gui/qgslayerpropertiesdialog.h

```diff
@@ -42,17 +42,33 @@ class GUI_EXPORT QgsLayerPropertiesDialog : public QgsOptionsDialogBase SIP_ABST
 
   public:
 
+#ifndef SIP_RUN
+
+    /**
+     * Style storage type.
+     */
+    enum StyleType
+    {
+      QML,
+      SLD,
+      DB,
+      Local,
+    };
+    Q_ENUM( StyleType )
```

Comment:
Let's clear up the naming here a bit for clarity: (This class isn't exposed to SIP, so we're free to rename)
```suggestion
    enum class StyleType
    {
      QML,
      SLD,
      RemoteDatabase,
      LocalDatabase,
    };
    Q_ENUM( StyleType )
```

Reply:
What about `DB` => `DatasourceDatabase`, `Local` => `UserDatabase`?